README.md

```diff
@@ -209,6 +209,26 @@ regardless of the authentication mechanism.
         - uses: 'google-github-actions/auth@v0'
      ```
 
+-   `export_environment_variables`: (Optional) If true, the action will export
```

Comment:
Should we call out this defaults to true?

Reply:
Done.